### PR TITLE
Daos summary

### PR DESCRIPTION
**Pull request by nikhilnanal**

### CODE REVIEW

**Review by zachdworkin — CHANGES_REQUESTED** [review]
Looks good, some minor nitpicks and one change necessary

File: contrib/intel/jenkins/summary.py [hunk 1]
```diff
@@ -436,6 +436,38 @@ def check_line(self, line):
         self.check_fail(line)
         super().check_exclude(line)
 
+class DaosSummarizer(Summarizer):
```
[the remark this answers is not in the document]
Comment (nikhilnanal) [reply contrib/intel/jenkins/summary.py:439]: there was one line which was exceeding. fixed that.

File: contrib/intel/jenkins/summary.py [hunk 2]
```diff
@@ -447,7 +479,7 @@ def check_line(self, line):
     parser = argparse.ArgumentParser()
     parser.add_argument('--summary_item', help="functional test to summarize",
                          choices=['fabtests', 'imb', 'osu', 'mpichtestsuite',
-                         'oneccl', 'shmem', 'ze', 'multinode', 'all'])
```
[the remark this answers is not in the document]
Comment (nikhilnanal) [reply contrib/intel/jenkins/summary.py:450]: added

[the remark this answers is not in the document]
Comment (nikhilnanal) [reply contrib/intel/jenkins/summary.py:450]: added

File: contrib/intel/jenkins/Jenkinsfile [hunk 3]
```diff
@@ -655,6 +660,21 @@ pipeline {
...
+    stage ('Summary-daos') {
+      agent {node {label 'daos'}}
+      when { equals expected: 1, actual: DO_RUN }
+      steps {
+        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH']) {
+          sh """
+            env
+            (
+              python3.7 ${env.WORKSPACE}/${SCRIPT_LOCATION}/summary.py --summary_item=daos
+              python3.7 ${env.WORKSPACE}/${SCRIPT_LOCATION}/summary.py -v --summary_item=daos
```
[the remark this answers is not in the document]
Comment (nikhilnanal) [reply contrib/intel/jenkins/Jenkinsfile]: put it in the success stage for daos.

**Review by zachdworkin — APPROVED** [review]
Summary in jenkins report looks good!

File: contrib/intel/jenkins/summary.py [hunk 4]
```diff
@@ -447,7 +481,7 @@ def check_line(self, line):
...
+                         'oneccl', 'shmem', 'ze', 'mutlinode', 'daos', 'all'])
```
Comment (zachdworkin) [inline contrib/intel/jenkins/summary.py:484]: multinode is misspelled

**Review by zachdworkin — CHANGES_REQUESTED** [review]
Can you change your commit messages from "Intel CI:" -> "contrib/intel/jenkins:" to be consistent?

There is also one typo that needs to be fixed before we can merge this.

File: contrib/intel/jenkins/summary.py (hunk 4 above)
Comment (nikhilnanal) [reply contrib/intel/jenkins/summary.py]: corrected!

**Review by zachdworkin — APPROVED** [review]

**Pull request merged**